packages/fast-usdc-contract/src/exos/advancer.ts

```diff
@@ -128,21 +127,18 @@ export const prepareAdvancerKit = (
   });
   const feeTools = makeFeeTools(feeConfig);
   /** @param {bigint} value */
-  const toAmount = value => AmountMath.make(usdc.brand, value);
+  const toAmount = (value: bigint) => AmountMath.make(usdc.brand, value);
 
   return zone.exoClassKit(
     'Fast USDC Advancer',
     AdvancerKitI,
-    /**
-     * @param {{
-     *   notifier: import('./settler.js').SettlerKit['notifier'];
-     *   borrower: LiquidityPoolKit['borrower'];
-     *   poolAccount: HostInterface<OrchestrationAccount<{chainId: 'agoric'}>>;
-     *   settlementAddress: CosmosChainAddress;
-     *   intermediateRecipient?: CosmosChainAddress;
-     * }} config
-     */
-    config =>
+    (config: {
+      notifier: any;
+      borrower: LiquidityPoolKit['borrower'];
+      poolAccount: HostInterface<OrchestrationAccount<{ chainId: 'agoric' }>>;
```

Comment:
`chainId: 'agoric'`? not `agoric-3`? odd. I don't think I understand how that works.

Reply:
the matcher works with either,
```
      ? CI['chainId'] extends `agoric${string}`
```

but I'll change this to `agoric-any` like the other time this came up